tests/recipes/test_full_finetune_distributed.py

```diff
@@ -29,7 +29,10 @@
 
 
 class TestFullFinetuneDistributedRecipe:
-    def _get_test_config_overrides(self):
```

Comment:
Both "optimizer_in_bwd=True" and "clip_grad_norm=100" could cause the wrong grad_norm, separate them here to avoid, loss_value would not be affected by either "optimizer_in_bwd=True" or "clip_grad_norm=100"